twilight-model/src/channel/message/call.rs

```diff
@@ -0,0 +1,16 @@
+use crate::{
+    id::{marker::UserMarker, Id},
+    util::Timestamp,
+};
+
+use serde::{Deserialize, Serialize};
+
+/// Information about the call in a private channel.
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct MessageCall {
+    /// The timestamp of when the call ended.
+    pub ended_timestamp: Option<Timestamp>,
```

Comment:
discord documents this as a nullable optional field, shouldnt it be `Option<Nullable<Timestamp>>`

Reply:
I believe this would only be applicable if this was a field that bot/app could edit, since it's readonly it being an `Option` should suffice.

Reply:
there might be a different reason behind this, it being `None` and just not existing could signify different things like one meaning the call is ongoing and another meaning its unknown

im not sure about what exactly it is for but i think we should follow discord docs here as we do in the rest of the library

Reply:
I will probably leave this as-is. I searched through everything in twilight-model and I can't seem to find anywhere where `Nullable` is being used. It seems to be exclusively used in the http crate.